lib/test.py

```diff
@@ -8,6 +8,7 @@
 import sys
 import traceback
 from functools import partial
+from hashlib import md5
```

Comment:
I don't want to nitpick to much, but the commit header 'Add ability to check failed tests w/ fragile list' does not give the idea what the commit offers. It is like 'everything as before, but with *check*'. 'check' may be almost anything for one, who doesn't read the patch itself.

Reply:
Corrected.